### PR TITLE
Update admixtools to 7.0.1

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`admixtools`](https://bioconda.github.io/recipes/admixtools/README.html): **6.0** &rarr; **7.0.1**

[install with bioconda](http://bioconda.github.io/recipes/admixtools/README.html) [Conda](https://anaconda.org/bioconda/admixtools/files)

Info | Link
-----|-----
Recipe | [`recipes/admixtools`](https://github.com//bioconda/bioconda-recipes/tree/bump/admixtools/recipes/admixtools) (click to view/edit other files)
Releases |[https://github.com/DReichLab/AdmixTools/tags](https://github.com/DReichLab/AdmixTools/tags)
Author | `@DReichLab`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Review by bwlang — APPROVED** [review]

**Comment by bwlang** [comment]
looks like this update is missing some build requirements...
```
08:21:17 BIOCONDA INFO (OUT) cc -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/admixtools-7.0.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -c -g -p -Wimplicit -I./ -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -isystem $PREFIX/include  -c -o strsubs.o strsubs.c
08:21:17 BIOCONDA INFO (OUT) make[1]: cc: Command not found
```

**Review by bwlang — CHANGES_REQUESTED** [review]
did not mean to approve this yet... looks like this will need some build updates to work.

**Review by rpetit3 — APPROVED** [review]

**Comment by rpetit3** [comment]
@bioconda-bot please merge

**Comment by rpetit3** [comment]
@bioconda-bot please fetch artifacts

**Review by rpetit3 — APPROVED** [review]

**Comment by rpetit3** [comment]
@bwlang Looks like you need to approve the changes to push this forward.

**Comment by rpetit3** [comment]
Pinging @bwlang again

File: recipes/admixtools/makefile-compiler.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+From 3fd1db48c11837bea0225037ec4e4fec8ae667ee Mon Sep 17 00:00:00 2001
+From: "Robert A. Petit III" <robbie.petit@gmail.com>
```
Comment (bwlang) [inline recipes/admixtools/makefile-compiler.patch:2]: Are you sure you want this detail in here?

**Review by bwlang — APPROVED** [review]

**Comment by bwlang** [comment]
> Pinging @bwlang again

Thanks for the reminder... not sure this needed me specifically - just some warm body (brain???)

**Comment by rpetit3** [comment]
Thanks @bwlang ! Haha yeah it needed your approval since you requested a change back in September.

**Comment by rpetit3** [comment]
@BiocondaBot please merge

**Pull request merged**